### PR TITLE
Some Changes In Bloodmoon

### PR DESCRIPTION
**Pull request by Ultradragon005**

### CODE REVIEW

File: Roles/(Ghosts)/Impostor/Bloodmoon.cs [hunk 1]
```diff
@@ -94,12 +93,13 @@ public override void OnOtherTargetsReducedToAtoms(PlayerControl target)
...
-    private string OthersNameText(PlayerControl seer, PlayerControl player, bool IsForMeeting) 
+    private string OthersNameText(PlayerControl seer, PlayerControl player, bool IsForMeeting, bool isforhud = false) 
     {
-        if (IsForMeeting || seer != player) return string.Empty;
+        var IsMeeting = GameStates.IsMeeting;
+        if (IsMeeting || seer != player || player == null || isforhud) return string.Empty;
         var playerid = player.PlayerId;  
 
-        if (LastTime.TryGetValue(playerid, out var lastTime) && lastTime + 1 <= GetTimeStamp() && !IsForMeeting)
+        if (LastTime.TryGetValue(playerid, out var lastTime) && lastTime + 1 <= GetTimeStamp() && !IsMeeting)
```
Comment (Tommy-XL) [inline Roles/(Ghosts)/Impostor/Bloodmoon.cs:102]: text in `isForHud` will not be displayed for modded players

And `GameStates.IsMeeting` doesn't work correctly in NotifyRoles, so better use `IsForMeeting`

Comment (Ultradragon005) [reply Roles/(Ghosts)/Impostor/Bloodmoon.cs]: When I used IsForMeeting, the target got killed during meeting..

I think it's maybe bcuz it returns false here? 
image

Comment (Tommy-XL) [reply Roles/(Ghosts)/Impostor/Bloodmoon.cs]: Then use `isForMeeting || GameStates.IsMeeting` and return `string.Empty`

Comment (Ultradragon005) [reply Roles/(Ghosts)/Impostor/Bloodmoon.cs]: done

**Pull request merged**